ansible/roles/provision-files/vars/main.yml

```diff
@@ -71,15 +71,15 @@ girder_file_downloads:
       sha512: e6b382f5ba3d4d7f17d29caa3f6ea06dd5db24fee3bf59b12ff748d58b1b064c374741c805ee7c07a2a6950761a38e5398f3707ed3e83d7d53218972c57d12a2
       dest: "{{ stage_dirs.object_detector }}/m2_det.pt"
     # Activity classifier
-    - file_id: 671bff2983e1acbfd87e723f
-      sha512: 5165219f60e3a160a8ec51f0446db4ec23ed70c399df66f482a233566e177d1f8cf044fc6ccc5227f5c748cbe6d2f17143109e791a139dfc2225d75de67829b2
```

Comment:
https://data.kitware.com/#item/671bff2983e1acbfd87e723e

Reply:
New: https://data.kitware.com/#item/673f69bef4ba555b2ab8ae14

Reply:
By "comments" I was at least meant to refer to comments in the actual file. @cameron-a-johnson is that what you meant as well, or this kind of github comment?